templates/taxi/car_detail.html

```diff
@@ -1,22 +1,45 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
+  <h1>Model: {{ car.model }}</h1>
+  <p><strong>Manufacturer:</strong> ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
+  
+  <h2>Drivers</h2>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Username</th>
+        <th>Full Name</th>
+        <th>License Number</th>
+      </tr>
+    </thead>
+    <tbody>
+      {% for driver in car.drivers.all %}
+        <tr>
+          <td>{{ driver.username }}</td>
+          <td>{{ driver.first_name }} {{ driver.last_name }}</td>
+          <td>{{ driver.license_number }}</td>
+        </tr>
+      {% endfor %}
+    </tbody>
+  </table>
+
+  <div class="btn-group" role="group" aria-label="Car actions">
+    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger">
       Delete
     </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
+    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary">
       Update
     </a>
-  </h1>
-  <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
-  <hr>
-  <ul>
-    {% for driver in car.drivers.all %}
-      <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
-    {% endfor %}
-  </ul>
+
+    {% if user in car.drivers.all %}
+      <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger">
+        Delete me from this car
+      </a>
+    {% else %}
+      <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-success">
+        Assign me to this car
+      </a>
```

Comment:
The URL for assigning the user to the car should point to the `assign_user_to_car` view, not `car-update`. Update the URL to `{% url 'taxi:car-assign' pk=car.id %}` to correctly handle the action.